### DISCUSSION TITLE
Does the UI Kit require a Token Studio Pro license?

### DISCUSSION THREAD

**Discussion by AnikaBausom** [primefaces/community#3144] [Figma UI Kit]
I just started a new UX Design job and my engineers like using PrimeNG. I was thinking about getting the UI Kit for Figma. It says it's compatible with Token Studio, but does that mean Token Studio is required? Do we need the Pro version? (It looks like the cost of the UI Kit went up a lot this week, and I'm not sure I can get budget approval for both.)

**Comment by onursenture** [comment]
Hello, you don't need Tokens Studio Pro to use the UI Kit with Tokens Studio.
